### PR TITLE
PR fix: Add test for null attribute

### PR DESCRIPTION
**Pull request by mydea**
This adds on top of https://github.com/getsentry/rrweb/pull/65 and adds a test for this, as well as streamlining this check in the `transformAttribute` method.

### CODE REVIEW

File: packages/rrweb-snapshot/src/snapshot.ts [hunk 1]
```diff
@@ -242,32 +242,35 @@ export function transformAttribute(
   maskAllText: boolean,
   maskTextFn: MaskTextFn | undefined,
 ): string {
+  if (!value) {
```
Comment (Lms24) [inline packages/rrweb-snapshot/src/snapshot.ts:245]: l: should we change `value`'s type to `string | undefined` in the function signature to reflect that it apparently can be undefined?

**Review by Lms24 — APPROVED** [review]

File: packages/rrweb-snapshot/src/snapshot.ts (hunk 1 above)
Comment (mydea) [reply packages/rrweb-snapshot/src/snapshot.ts:245]: makes sense!

Comment (mydea) [reply packages/rrweb-snapshot/src/snapshot.ts:245]: hmm, it seems there is actually a longer tail to this (as a lot of types seem to depend on the return value of this being `string`.)

So IMHO it's correct, but let's look at this separately maybe then!
